vhdx/src/vhdx_header.rs

```diff
@@ -0,0 +1,536 @@
+extern crate log;
+
+use byteorder::{ByteOrder, LittleEndian, ReadBytesExt, WriteBytesExt};
+use std::collections::btree_map::BTreeMap;
+use std::fmt::{self, Display};
+use std::fs::File;
+use std::io::{self, Read, Seek, SeekFrom, Write};
+use std::mem::size_of;
+use uuid::Uuid;
+
+pub enum Error {
+    InvalidVHDXSign,
+    ReadingFileTypeIndetifier(io::Error),
+    InvalidHeaderSign,
+    ReadingHeader(io::Error),
+    InvalidRegionSign,
+    ReadingRegionTableHeader(io::Error),
```

Comment:
Use the verb infinitive form such as `ReadRegionTableHeader` or `WriteData`.

Reply:
Done.